setup.py

```diff
@@ -51,6 +51,9 @@ def pypi_description():
     with open('PyPiDescription.rst') as df:
         return df.read()
 
+def get_nightly_version():
+    today = date.today()
+    return f"{today.year}.{today.month}.{today.day}"
```

Comment:
nit (see https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior )
```suggestion
    return today.strftime("%Y.%m.%d")
```